src/kitovu/cli.py

```diff
@@ -27,10 +27,11 @@ def cli() -> None:
 
 
 @cli.command()
-@click.argument('url')
-def sync(url: str) -> None:
-    """Synchronize the given URL."""
+@click.argument('plugin')
+@click.argument('username')
+def sync(plugin: str, username: str) -> None:
+    """Synchronize with the given plugin and username."""
     try:
-        syncing.start(url)
+        syncing.start(plugin, username)
```

Comment:
so as we discussed, the plugin is thus identified via plugin+username (and not the URL concept)?

Reply:
The plugin is only identified by the plugin name (`plugin` here, which is e.g. `'smb'`). But on the commandline, you also pass the username (e.g. `kitovu sync smb msieber`) so it knows with what username to log in.

At a later stage, all that will be part of a config file, but that doesn't exist yet.